### PR TITLE
Fix tls option in odh-dashboard route

### LINKED ISSUE #1639: [Bug]: Wrong TLS Option in dashboard route
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Deploy type

OpenDataHub core version (eg. `v1.6.0`)

### Version

1.7.0

### Current Behavior

The current values for `route.openshift.io/v1/spec/tls/termination` are `edge`, `passthrough ` and `reencrypt`, if you give another value it will fail silently, we have `Reencrypt` as default value.

### Expected Behavior

To have the right value in our deployment.

### Steps To Reproduce

_No response_

### Workaround (if any)

_No response_

### What browsers are you seeing the problem on?

_No response_

### Anything else

_No response_

### PR DESCRIPTION
**Pull request by lucferbux**
## Description

Closes #1639 

## Request review criteria:

Self checklist (all need to be checked):
- [X] The developer has manually tested the changes and verified that the changes work
- [X] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change.

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

**Comment by shalberd** [comment]
nice catch, cool, lgtm

**Review by DaoDaoNoCode — APPROVED** [review]
lgtm

**Comment by lucferbux** [comment]
/hold

Let's wait until next release for this one.

File: manifests/base/routes.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ spec:
     targetPort: 8443
   tls:
     insecureEdgeTerminationPolicy: Redirect
-    termination: Reencrypt
+    termination: reencrypt
```
Comment (andrewballantyne) [inline manifests/base/routes.yaml:13]: Wild that the OpenShift folks chose lowercase for this but not for all properties -- I would imagine all of these are just magic strings.

But it is in the API Explorer as such
image

**Review by andrewballantyne — APPROVED** [review]
/unhold

Release was made, this can go into the next

LGTM.

**Comment by jbyrne-redhat** [comment]
@lucferbux How does this release note text sound?

### Wrong TLS value in dashboard route
Previously, when a route was created for the {productname-short} dashboard on OpenShift, the `tls.termination` field had an invalid default value of `Reencrypt`. This issue is now resolved. The new value is `reencrypt`.

**Pull request merged**